hack/build-docs.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+BIN=./func
```

Comment:
One nit, shouldn't we run automatically? add Makefile target for this to be part of `build`

Reply:
Yes I suppose so - I will take a look into that.